modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -13607,17 +13863,21 @@ private DLFolder _getDLFolder(
 	}
 
 	private String _getEndpoint(
-		long groupId, ObjectDefinition objectDefinition) {
+		String scopeKey, long groupId, ObjectDefinition objectDefinition) {
```

Comment:
The correct approach would be not to add a new parameter, but rather to change the existing `long groupId` parameter to `String scopeKey`, and adapt the current usages by converting groupId to `String.valueOf(groupId)`. I understand it’s a significant refactor, but I believe it’s the right choice. Could we proceed with this, please?

Reply:
I did something a little different by changing from long or String to `Object` since it's a type that accept the wrappers String and Long from the Java.